### PR TITLE
add navigation bar

### PR DESCRIPTION
**Pull request by manch0ffline**
[DEMO LINK](https://manch0ffline.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://manch0ffline.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Links are not working. Make sure that you've successfully run `npm run deploy`

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,25 @@
...
+      <nav>
+        <ul>
+          <li><a class="is-active link" href="#">Apple</a></li>
+          <li><a class="link" href="#">Samsung</a></li>
+          <li><a class="link" href="#">Smartphones</a></li>
+          <li><a data-qa="hover" class="link">Laptops & Computers</a></li>
+          <li><a class="link" href="#">Gadgets</a></li>
+          <li><a class="link" href="#">Tablets</a></li>
+          <li><a class="link" href="#">Photo</a></li>
+          <li><a class="link" href="#">Video</a></li>
```
Comment (SanyaBratashchuk) [inline src/index.html:26]: Fix formatting according to https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md code style rules

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
```
Comment (SanyaBratashchuk) [inline src/style.css:9]:
```suggestion
```
You have already applied it on line 4

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  align-items: center;
+  border: none;
+}
+
+a.link {
```
Comment (SanyaBratashchuk) [inline src/style.css:24]: do not increase css specificity via combined selectors without necessity 
```suggestion
link {
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,78 @@
...
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+li {
+  margin-left: 20px;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:54]: use css classes for styling purposes

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,41 @@
...
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <div class="img"></div>
+      </a>
+      <nav class="nav">
+        <ul class="nav__lists">
+          <li class="nav__list_link">
+            <a class="is-active link" href="#">Apple</a>
+          </li>
```
Comment (volodymyr-soltys97) [inline src/index.html:21]: Let's fix it everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="is-active nav__link" href="#">Apple</a>
          </li>
          
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,87 @@
...
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+header {
```
Comment (volodymyr-soltys97) [inline src/style.css:12]: Need to write styles with classes selector

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,87 @@
...
+@media screen and (min-width: 1024px) {
+  header {
+    width: 1024px;
+  }
 }
+
+@media screen and (min-width: 1200px) {
+  header {
+    width: 1200px; */
+  /* }
```
Comment (volodymyr-soltys97) [inline src/style.css:86]: Remove comments

File: src/index.html [hunk 8]
```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
```
Comment (volodymyr-soltys97) [inline src/index.html:13]:
```suggestion
    <header class="header">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,75 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Roboto:wght@400;700&display=swap");
```
Comment (maxim2310) [inline src/style.css]: you need to use Roboto 500 font weight 
image

File: src/style.css (hunk 2 above)
Comment (maxim2310) [reply src/style.css:5]: not fixed https://github.com/mate-academy/layout_moyo-header/pull/4104/files#r1393204350

**Comment by manch0ffline** [comment]
I can't understand the test problem

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,75 @@
...
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (maxim2310) [inline src/style.css:63]: need to use variable for this color

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! just try to pass all tests

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,74 @@
+html {
+  --color-text-hover: #00acdc;
+}
+
 body {
   margin: 0;
```
Comment (anastasiiavorobiova) [inline src/style.css:6]:
```suggestion
  margin: 0;
  font-family: "Roboto", sans-serif;
```
It's better to specify a font family here to get it inherited by other elements

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,74 @@
+html {
+  --color-text-hover: #00acdc;
+}
+
```
Comment (anastasiiavorobiova) [inline src/style.css:4]:
```suggestion
* {
  box-sizing: border-box;
}
```

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,74 @@
...
+.nav__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
```
Comment (anastasiiavorobiova) [inline src/style.css:38]:
```suggestion

```

File: src/style.css [hunk 14]
```diff
@@ -1,3 +1,74 @@
...
+  position: relative;
+}
+
+.nav__link:hover {
+  color: var(--color-text-hover);
+}
+
+.is-active::after {
+  content: "";
+  box-sizing: border-box;
```
Comment (anastasiiavorobiova) [inline src/style.css:66]:
```suggestion

```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by VitaliyBodnar — APPROVED** [review]

**Pull request closed without merging**